study-builder/studies/rarex/patches/patch_0721.conf

```diff
@@ -0,0 +1,483 @@
+                }
+              ]
+            }
+          ]
+        },
+        "detailLabelTemplate": null,
+        "allowDetails": false,
+        "exclusive": false
+      },
+      {
+        "stableId": "SINGLE_SERVICE_PLAN",
+        "optionLabelTemplate": {
+          "templateType": "TEXT",
+          "templateText": "$single_service_plan",
+          "variables": [
+            {
+              "name": "single_service_plan",
+              "translations": [
+                {
+                  "language": "en",
+                  "text": ${i18n.en.single_service_plan}
+                }
+              ]
+            }
+          ]
+        },
+        "detailLabelTemplate": null,
+        "allowDetails": false,
+        "exclusive": false
+      },
+      {
+        "stableId": "NO_COVERAGE",
+        "optionLabelTemplate": {
+          "templateType": "TEXT",
+          "templateText": "$no_coverage",
+          "variables": [
+            {
+              "name": "no_coverage",
+              "translations": [
+                {
+                  "language": "en",
+                  "text": ${i18n.en.no_coverage}
+                }
+              ]
+            }
+          ]
+        },
+        "detailLabelTemplate": null,
+        "allowDetails": false,
+        "exclusive": false
+      },
+      {
+        "stableId": "DONT_KNOW",
+        "optionLabelTemplate": {
+          "templateType": "TEXT",
+          "templateText": "$dont_know",
+          "variables": [
+            {
+              "name": "dont_know",
+              "translations": [
+                {
+                  "language": "en",
+                  "text": ${i18n.en.dont_know}
+                }
+              ]
+            }
+          ]
+        },
+        "detailLabelTemplate": null,
+        "allowDetails": false,
+        "exclusive": false
+      }
+    ]
+    "validations": [
+      {
+        "ruleType": "REQUIRED",
+        "allowSave": true,
+        "hintTemplate": {
+          "templateType": "TEXT",
+          "templateText": "$req_hint",
+          "variables": [
+            {
+              "name": "req_hint",
+              "translations": [
+                {
+                  "language": "en",
+                  "text": ${i18n.en.required_field_hint}
+                }
+              ]
+            }
+          ]
+        }
+      }
+    ],
+    "stableId": "GENERAL_INFORMATION_HIC_DETAIL"
+  },
+  "preferNotToAnswerCheckboxSelf": {
+    "question": {
+      include required("../../snippets/picklist-question-multi-list.conf"),
+      "hideNumber": true,
+      "promptTemplate": {
+        "templateType": "TEXT",
+        "templateText": "",
+        "variables": []
+      },
+      "picklistLabelTemplate": {
+        "templateType": "TEXT",
+        "templateText": "",
+        "variables": []
+      },
+      "picklistOptions": [
+        {
+          "stableId": "PREFER_NO_TO_ANSWER",
```

Comment:
Did we mean `PREFER_NOT_TO_ANSWER`?

Reply:
@YuriyPavlovBI :
Are you going to take care of this change to change the stableID to PREFER_NOT_TO_ANSWER ?

Reply:
Sure